### DISCUSSION TITLE
Craft 4 - Rich Text Link always has a blank target

### DISCUSSION THREAD

**Discussion by DowleyDeveloped** [solspace/craft-freeform#889] [Feature Requests]
**Describe the bug or issue you're experiencing**

When you include a link within the rich text field, the target is always '_blank' even when using a relative url. This should be an option within the popup.

**Steps to reproduce**

1. Add a Rich Text field and add a link
2. Click on the link via the front end, and it opens a new tab/window

**Expected behavior**

We should have the option of whether this opens a new tab/window or not.

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 4.5.6.1
- Freeform Version: 4.1.10
- Freeform Edition: Pro
- Fresh Install or Upgrade: Upgrade

**Comment by kjmartens** [comment]
Thanks for the feedback, @DowleyDeveloped. We have noted this for future consideration. I have a hunch this might be a bit trickier as it relies on a third-party feature.

**Comment by kjmartens** [comment]
It's been a long time, but just letting you know that we've upgraded the Rich Text editor in Freeform 5.10+ and you can now choose whether the link should open in the same window or a new one. 🙂
